src/elexsolver/QuantileRegressionSolver.py

```diff
@@ -78,7 +87,7 @@ def fit(self, x, y, tau_value=0.5, weights=None, verbose=False, save_problem=Fal
             weights = weights / weights_sum
 
         self.tau.value = tau_value
-        coefficients, problem = self.__solve(x, y, weights, verbose)
+        coefficients, problem = self.__solve(x, y, weights, lambda_, verbose)
```

Comment:
Do we want to have some sort of message or output that tells us if the resulting coefficients are 0 or within some tolerance of 0, to help us with feature selection?

Reply:
That's a good idea! Though I think it might be better to do that in the model repo? or then in the admin? since looking at the logs is a bit of a pain at the moment. What do you think?

Reply:
Hmm solver repo may make a little more sense, because it's so related to what the solver is actually doing? But either way, I think we def want the message to appear in local runs (outside of admin) for us. And then yes, because the logs are annoying, I agree for real election monitoring we also want it in admin -- wherever model results get updated with every run. No idea how to set up the solver-repo side of the admin piece here?

Reply:
ok, I'm going to add an attribute in the model that does that. I think that is better because the solver doesn't know what feature is what (we only pass a numpy array in, so it doesn't have access to the feature names)